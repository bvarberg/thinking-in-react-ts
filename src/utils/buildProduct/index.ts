import { Product } from "../../types/Product"

type BuildProduct = () => Product

export const buildProduct: BuildProduct = () => {
  return {
    category: "Sporting Goods",
    name: "Football",
    price: "$49.99",
    stocked: true,
  }
}
