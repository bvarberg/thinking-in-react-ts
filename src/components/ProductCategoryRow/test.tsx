import { ProductCategoryRow } from "."
import React from "react"
import { shallowRender } from "../../utils/shallowRender"

describe("ProductCategoryRow", () => {
  it("renders", () => {
    const category = "Sporting Goods"

    const received = shallowRender(<ProductCategoryRow category={category} />)

    expect(received).toMatchSnapshot()
  })
})
