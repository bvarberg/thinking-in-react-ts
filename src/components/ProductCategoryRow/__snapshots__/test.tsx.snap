// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCategoryRow renders 1`] = `
<tr
  className="ProductCategoryRow"
>
  <th
    colSpan="2"
  >
    Sporting Goods
  </th>
</tr>
`;
